Lib/enum.py

```diff
@@ -162,6 +162,13 @@ def _dedent(text):
         lines[j] = l[i:]
     return '\n'.join(lines)
 
+class _not_given:
+    def __repr__(self):
+        return('<not given>')
+    def __bool__(self):
+        return False
```

Comment:
Is it needed?

Reply:
Yes, or the `or None` at line 730 will fail.

Reply:
I'm not sure what you mean -- is "second value" the names parameter?  If it's falsey and a type is given, a new empty enum is created (which is appropriate).